CHANGELOG.md

```diff
@@ -1,5 +1,69 @@
 # Database Tools Changelog
 
+## 100.7.0
+
+_Released 2023-03-01_
+
+We are pleased to announce version 100.7.0 of the MongoDB Database Tools.
+
+This release adds tests against MongoDB 6.3. Highlights include new tests for [Column Store Indexes](https://www.mongodb.com/products/column-store-indexes), updating Go version to 1.19, fixing a bug that caused the Tools to produce error after prompting for a password. Several build failures are also fixed in this version.
```

Comment:
nit: "updating _the minimum_ Go version" - this repo is FOSS so others can compile with a later Go.

Also: "caused the Tools to produce error after prompting for a password" isn't quite right. I'd say "fixing a bug that caused the Tools to ignore a password supplied via a prompt".